client/src/components/Hero.css

```diff
@@ -0,0 +1,62 @@
+:root {
+	font-size: 16px;
+}
+
+.hero-container {
+	height: 26.2em;
```

Comment:
This is fine, but generally when using `em` and a 16px font, we choose multiples of 0.125 (2px), 0.25 (4px), 0.5 (8px). These will resolve to whole px numbers. This may not result in "pixel perfect" replications of the designs, but it is more readable and more visually consistent.

No changes needed here but this is an informational comment. PX values should be whole numbers, EM should be certain fractions that result in whole PX numbers.